07-routing-gossip.md

```diff
@@ -262,6 +262,18 @@ nodes not associated with an already known channel are ignored.
    * [`32*byte`:`alias`]
    * [`u16`:`addrlen`]
    * [`addrlen*byte`:`addresses`]
+   * [`node_ann_tlvs`:`tlvs`]
+
+1. `tlv_stream`: `node_ann_tlvs`
+2. types:
+   1. type: 1 (`option_will_fund`)
+   2. data:
+       * [`u16`:`funding_fee_proportional_basis`]
+       * [`u32`:`funding_fee_base_sat`]
+       * [`u16`:`funding_weight_max`]
+       * [`u16`:`channel_fee_proportional_basis_max`]
+       * [`tu32`:`channel_fee_base_max_msat`]
```

Comment:
Use subtype `lease_rates` directly? Yes, I know this creates a dep between the two BOLTs, but that's OK, and if it's not we fix our own tooling :)

Reply:
Oh, and it means we need to put signature field first in BOLT 2, since tu32 must be at end of TLV.